File: SoftwareTechnologies/JS/JSdemo/Largest3Numbers.js
function Largest3Numbers(arr) {
    let sorted = arr.sort((a, b) => a - b);

    let length = sorted.length > 2 ? 3 : sorted.length;

    for (let i = 0; i < length; i++){
        console.log(sorted.pop());
    }
}

Largest3Numbers([10, 30])

File: SoftwareTechnologies/Java/JavaLab/Largest3Numbers.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.Arrays;
import java.util.stream.Collectors;

public class Largest3Numbers {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        Arrays.stream(reader.readLine().split(" "))
                .map(Integer::parseInt)
                .sorted((e1, e2) -> Integer.compare(e2, e1))
                .limit(3)
                .collect(Collectors.toList())
                .forEach(System.out::println);

    }
}
